Library/Formula/poppler.rb

```diff
@@ -51,8 +53,12 @@ def install
 
     if build.with? "qt"
       args << "--enable-poppler-qt4"
+    elsif build.with? "qt5"
+      ENV.prepend_path "PKG_CONFIG_PATH", "#{Formula["qt5"].opt_prefix}/lib/pkgconfig"
```

Comment:
It works for qt4 but not qt5 for some reason. I have not looked into the underlying reason for it, just added this line to get Poppler to build properly. I see several other formulas adding explicit PKG_CONFIG_PATH entries in a similar way.